### PR TITLE
add filter function in processor

### LINKED ISSUE #13: Wrapper function to filter data frame
Partially implemented in hextof-processor

### PR DESCRIPTION
**Pull request by steinnymir**
add the functionality to filter the dataframe to a range of values.

closes #13

### CODE REVIEW

**Comment by steinnymir** [comment]
@rettigl @zain-sohail this is a simple addition. Ready for review.

**Review by zain-sohail — APPROVED** [review]
Useful addition! Works fine when I tested

File: sed/core/processor.py [hunk 1]
```diff
@@ -402,6 +403,46 @@ def load(
...
+                lower_bound=min_value,
+                upper_bound=max_value,
+            )
+            metadata = {
+                "filter": {
+                    "column": column,
+                    "min_value": min_value,
+                    "max_value": max_value,
+                },
+            }
```
Comment (rettigl) [inline sed/core/processor.py:443]: this should be outside the if-clause, no?

Comment (steinnymir) [reply sed/core/processor.py:443]: yes! fixed!

**Pull request merged**